components/builder-api/habitat/default.toml

```diff
@@ -34,4 +34,7 @@ bucket_name = "habitat-builder-artifact-store.default"
 
 [upstream]
 endpoint = "http://localhost"
-origins = ["core"]
\ No newline at end of file
+origins = ["core"]
+
+[memcache]
+ttl = 15
```

Comment:
Minor- I'm thinking we should default ttl to much smaller - eg 1 min

Reply:
Yeah.. it's just easier to recover any missed cases with a smaller ttl... however having a larger one may help us flush out the missed cases...so yeah either is ok

Reply:
It's got a lever in the config so we should be able to tweak it from there as needed then update the default accordingly.